src/main/java/dtu/scheduler/SchedulingApp.java

```diff
@@ -1,12 +1,22 @@
 package dtu.scheduler;
 
+import java.util.List;
+//Philip Hviid
 public class SchedulingApp {
 	private Worker currentUser;
-	private WorkerDAO workerDAO = new WorkerDAO();
+	private WorkerDAO workerDAO;
+	private ActivityAssigner activityAssigner;
+	private AssistRequestHandler requestHandler;
+	
+	public SchedulingApp(WorkerDAO workerDAO, ActivityAssigner activityAssigner, AssistRequestHandler requestHandler) {
```

Comment:
Seems pointless since we don't ever construct a non-default SchedulingApp object
Will probably fix this in refactoring later if you leave it in anyway